.gitignore

```diff
@@ -1,3 +1,5 @@
 # Need a config/environment.rb for rails.vim to work in engines
 config/environment.rb
 *gem
+.idea
+.bundle
```

Comment:
Seems like an unnecessary change? There's no reason to run bundler from within a Foreman plugin.